README.md

```diff
@@ -37,8 +37,9 @@ module.exports = {
   reporters: ['dot', 'junit'],
   reporterOptions: {
     outputDir: './',
-    outputFileFormat: function(opts) { // optional
-        return `results-${opts.cid}.${opts.capabilities}.xml`
+    outputFileFormat: function(opts) {
+      var suiteName = opts.config.suite.replace(/([^a-z0-9]+)/gi, '-');
```

Comment:
I think it would make more sense to sanitize the argument, so the user doesn't need to do that by himself